package/crds/ec2.aws.crossplane.io_addresses.yaml

```diff
@@ -18,46 +18,6 @@ spec:
     singular: address
   scope: Cluster
   versions:
-  - name: v1alpha1
```

Comment:
Did we expect these versions to disappear?

Reply:
@muvaf same question - in a few ec2 crds stuff is removed is this expected ? the rest in this PR looks good - tested also a few resources

Reply:
Yes, we added the `skipversion` tags to get those removed. See https://github.com/crossplane/provider-aws/issues/876 for details.